node_modules/@okta/jquery.simplemodal/package.json

```diff
@@ -20,13 +19,14 @@
     "type": "version"
   },
   "_requiredBy": [
+    "/",
```

Comment:
not sure this is right, since simplemodal is only requred by courage. 
But if Travis is passing it should be ok... at the end / means it could be required by whoever.